yarn.lock

```diff
@@ -2534,20 +2554,20 @@
   dependencies:
     "@octokit/openapi-types" "^9.1.0"
 
-"@openmrs/esm-api@^3.1.10-pre.430":
-  version "3.1.10-pre.430"
-  resolved "https://registry.yarnpkg.com/@openmrs/esm-api/-/esm-api-3.1.10-pre.430.tgz#f20132641e332d91e42ea5ff1012b2901ca4de68"
-  integrity sha512-LPwSNDGGizaoCJIHDDekGZ0YriQcsmrBbSbE2kRHOWMohZLnCY/S/2TBkk9UuoTOcgDkaq48gv+PhAFjUdqG/w==
+"@openmrs/esm-api@^3.1.10-pre.430", "@openmrs/esm-api@^3.1.10-pre.457":
```

Comment:
Hm shouldn't be. There should be only one version of `esm-api` (or other framework packages) demanded.